ui/src/Auth/TokenProvider.tsx

```diff
@@ -28,16 +31,26 @@ export const removeToken = (): void => {
     cookie.remove('accessToken', {path: '/'});
 };
 
-export const getDecodedToken = (): unknown | null  => {
+export const getDecodedToken = (): unknown | null => {
     const accessToken = getToken();
     return jwtDecoder(accessToken);
 };
 
+const addUserToMatomo = (userName: string) => {
```

Comment:
Should this method be in Matomo events?

Reply:
This only happens once, so probably not necessary?

Reply:
Mike and I agree it makes more sense to keep all the matomo logic together if we can. I'd hate for someone to need this method again and not realize it already exists and where it exists.